autogen.sh

```diff
@@ -46,6 +46,18 @@ if [ "$1" = "clean" ]; then
     find . -iname "Makefile.in" -type f -exec rm '{}' +
 fi
 
+# Prevent any errors that might result from failing to properly invoke `libtoolize` or `glibtoolize,` whichever 
+# is present on your system, from occurring by testing for its existence and capturing the absolute path to its 
+# location for caching purposes prior to using it later on in 'Step 2:'  
+if command -v libtoolize >/dev/null 2>&1; then
```

Comment:
Is it necessary to redirect stdout and stderr? As far as I see it is not needed (and so it should be removed).